src/utils/embedded/embedded.provider.tsx

```diff
@@ -1080,6 +1086,17 @@ export function EmbeddedProvider({ children }: EmbeddedProviderProps) {
 
       let authStatus = "noAuth" as AuthStatus;
 
+      // Check for passkey authentication (should be flagged in localStorage)
+      const isPasskeyAuth = embeddedContextAuth.authProviderType === "PASSKEYS";
+      const isCustomAuth = localStorage.getItem("isCustomAuth") === "true";
```

Comment:
Why is this needed? Would passkeys still work if the user cleans the browser data?